senders/http_proxy_sender.go

```diff
@@ -0,0 +1,62 @@
+package senders
+
+import (
+	"bytes"
+	"crypto/tls"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	interfaces "github.com/shoplineapp/captin/interfaces"
+	models "github.com/shoplineapp/captin/models"
+	log "github.com/sirupsen/logrus"
+)
+
+var hLogger = log.WithFields(log.Fields{"class": "HTTPProxyEventSender"})
+
+// HTTPResponse - HTTP Response
+type HTTPResponse struct {
+	url      string
+	response *http.Response
+	err      error
+}
+
+// HTTPProxyEventSender - Send Event through HTTP with payload only
+type HTTPProxyEventSender struct {
+	interfaces.EventSenderInterface
+}
+
+// SendEvent - #HTTPProxyEventSender SendEvent
+func (c *HTTPProxyEventSender) SendEvent(e models.IncomingEvent, d models.Destination) error {
+	url := d.GetCallbackURL()
+	payload, err := json.Marshal(e.Payload)
```

Comment:
Instead of parsing the whole event for event target type/id, we only need payload for third party api calls.